### PR TITLE
Log model

### LINKED ISSUE #53: Create Log Model

### PR DESCRIPTION
**Pull request by Marcellia**
closes #53 : Creating Log Model

### CODE REVIEW

File: src/app/models/LogModel.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+export interface LogInterface {
+  id: mongodb.ObjectId;
+  user: string;
+  action: string;
+  createdAt: Date;
+}
+
+export class LogModel implements LogInterface {
+  id: mongodb.ObjectId;
```
Comment (jpatrici) [inline src/app/models/LogModel.ts]:
```suggestion
  _id: mongodb.ObjectId;
```

File: src/app/models/LogModel.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  user: string;
+  action: string;
+  createdAt: Date;
+
+  constructor(
+    user = '',
+    action = '',
+    createdAt = new Date(),
+  ) {
+    this.id = new mongodb.ObjectId();
```
Comment (jpatrici) [inline src/app/models/LogModel.ts]:
```suggestion
    this._id = new mongodb.ObjectId();
```

File: src/app/models/LogModel.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import * as mongodb from 'mongodb';
+
+
+export interface LogInterface {
+  id: mongodb.ObjectId;
```
Comment (jpatrici) [inline src/app/models/LogModel.ts]:
```suggestion
  _id: mongodb.ObjectId;
```

**Review by jpatrici — APPROVED** [review]
i think it should be `_id` instead of `id`, but otherwise looks good!

File: src/app/models/LogModel.ts (hunk 1 above)
Comment (Marcellia) [reply src/app/models/LogModel.ts]: Good catch!

**Review by theodorusandi — APPROVED** [review]

**Pull request merged**